internal/controllers/sqlitecontroller/sqlitecontroller.go

```diff
@@ -0,0 +1,63 @@
+package sqlitecontroller
+
+import (
+	"database/sql"
+	"log"
+
+	_ "github.com/mattn/go-sqlite3"
+)
+
+// current implementation is for sqlite
+
+func SelectStudentByID(id string) (string, error) {
+	// try opening database; if failed, throw errow
+	db, err := sql.Open("sqlite3", "./build/studtor.db")
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer db.Close()
+
+	stmt, err := db.Prepare("SELECT student_name FROM students where student_id = ?")
```

Comment:
not sure it would be good to write raw sql queries actually. perhaps we should use an ORM such as [GORM](https://gorm.io/index.html)?

Reply:
Implemented in PR #19